src/components/Popover/Popover.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Popover } from "./Popover";
+
+const PopoverComponent = ({
+  open,
+  modal,
+  showArrow,
+  showClose,
+  forceMount,
+}: {
+  open: "default" | "open" | "closed";
+  modal: boolean;
+  showArrow: boolean;
+  showClose: boolean;
+  forceMount?: boolean;
+}) => (
+  <Popover
+    open={open === "default" ? undefined : open === "open"}
+    modal={modal}
+  >
+    <Popover.Trigger>
+      <div>Click here</div>
+    </Popover.Trigger>
+    <Popover.Content
+      showArrow={showArrow}
+      showClose={showClose}
+      forceMount={forceMount ? true : undefined}
+    >
+      Content popover
```

Comment:
lets try to add a more real world example with other components inside, like a few checkboxes and a form field to setup some name. with a save and cancel button.

This will give us a better idea on how to use this and might show some issues with our current styling.